<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>darkwindow</class>
 <widget class="QMainWindow" name="darkwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1321</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dark Wallet</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <item row="2" column="2">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="bottomMargin">
            <number>12</number>
           </property>
           <item>
            <widget class="QLineEdit" name="amount_lineedit">
             <property name="text">
              <string/>
             </property>
             <property name="placeholderText">
              <string>Amount</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="send_button">
             <property name="text">
              <string>&amp;Send</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="0" rowspan="3">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="balance_label">
             <property name="font">
              <font>
               <pointsize>60</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>20.424</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="currency_label">
             <property name="font">
              <font>
               <pointsize>20</pointsize>
               <italic>false</italic>
              </font>
             </property>
             <property name="text">
              <string>BTC</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
             </property>
             <property name="margin">
              <number>11</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="2">
          <widget class="QCheckBox" name="checkBox">
           <property name="text">
            <string>Mixnet</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLineEdit" name="destination_lineedit">
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Destination</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QTextEdit" name="output_log">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="palette">
          <palette>
           <active>
            <colorrole role="Text">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>92</red>
               <green>213</green>
               <blue>17</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Base">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>30</red>
               <green>26</green>
               <blue>26</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="Text">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>92</red>
               <green>213</green>
               <blue>17</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Base">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>30</red>
               <green>26</green>
               <blue>26</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="Text">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>151</red>
               <green>153</green>
               <blue>151</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Base">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>249</red>
               <green>250</green>
               <blue>249</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="font">
          <font>
           <family>Monospace</family>
          </font>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QFrame" name="frame_3">
      <property name="styleSheet">
       <string notr="true">QLabel { background-color : black; }</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QTableWidget" name="commitment_table">
         <property name="columnCount">
          <number>2</number>
         </property>
         <row>
          <property name="text">
           <string>1</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>Commitment</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Timestamp</string>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string>030c5a612efb0f5eb25eda5a3bff84458b9368b2d0fc46eb6cc7fff598cb3723c2</string>
          </property>
         </item>
         <item row="0" column="1">
          <property name="text">
           <string>6 Dec 2018</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/newPrefix/5c1215be3185eaec3e73c136_cyphersquatt.svg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="margin">
          <number>10</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="darktech.qrc"/>
 </resources>
 <connections/>
</ui>
